frontend/src/static/js/main.js

```diff
@@ -237,7 +237,7 @@ window.app = new window.Vue({
       if (Object.keys(info).length === tabInfoLength) {
         this.updateTabAuthorship(info);
       } else if (hash.tabOpen === 'false' || tabInfoLength > 2) {
-        window.app.isTabActive = false;
+        this.isTabActive = false;
```

Comment:
is this change needed? Also, what was the cause of the previous issue?

Reply:
> hmm, should here also remove tabAuthor, tabRepo and authorshipIsMergeGroup instead of removing in main.js?

still not fixed, this actually exposed a race condition I haven't thought of before.
I may need to move all hashes (including ones introduced here) back to `main.js` - still thinking if there are better solutions though

Reply:
sorry false alarm 😞

the issue was simply that I didn't move `setInfoHash` into/after `retrieveHashes` as well per my comment here https://github.com/reposense/RepoSense/pull/1057#issuecomment-604363267, which was the 'race condition' 🤦‍♂️ 
( it causes the hashes to be removed after `setInfohash` is called, which in turn causes `window.app.isTabActive = false` above to execute - hence the tab disappears  )